crates/sui-framework/sources/balance.move

```diff
@@ -101,6 +103,16 @@ module sui::balance {
         aborts_if balance.value != 0 with ENonZero;
     }
 
+    /// Can only be called by `sui_system` to create staking rewards.
+    public(friend) fun create<T>(value: u64): Balance<T> {
```

Comment:
Maybe make the two names very explicit to indicate they are special and should not be called otherwise, and if called should not be changed without careful thought

Reply:
Changed the names and added some comments